slick-testkit/src/test/scala/slick/test/lifted/QueryMethodTest.scala

```diff
@@ -43,5 +43,13 @@ class QueryMethodTest {
     println(s6)
     assertTrue("filterIf stacks", s6 endsWith """where ("myString" = 'stack') and ("myString" like '%yes' escape '^')""")
 
+    val s7 = ts.filterIf(true){ d => (d.id,d.price) inSet Seq(1->1.0, 2->2.0)}.filterIf(true)(_.myString === "stack").result.statements.head
```

Comment:
Not sure if this is the right place for these tests. As these are cases for the `filterIf` and `filterOpt` methods.

The `inSet` and `inSetBind` methods seem to be tested as part of the `MainTest` test class. Maybe you can include these testcases there.

Reply:
not All Database support  sql in with multiple columns. Maybe I should  add this as capability to compatibilities and test it in MainTest

Reply:
Sounds like a good idea!